packages/create-remix/utils.ts

```diff
@@ -266,3 +268,26 @@ export function action(key: ActionKey, isSelect: boolean) {
 
   return false;
 }
+
+export async function getDirectoryFilesRecursive(d: string) {
+  // Ignore comparing within these directories - but detect a collision
+  // at the directory level and count it.  These get prepended to strippedFiles
+  // in reverse order below
+  let ignoreDirs = ["node_modules", ".git"];
```

Comment:
We don't check file-by-file inside `.git`/`node_modules` but we treat them as collisions regardless.  There's a part of me that wants to add a hard stop on ever allowing an overwrite of `.git/` though...Thoughts?

Reply:
Yeah, I agree with this sentiment, overwriting `.git` feels pretty much guaranteed to be a mistake.

Rather than being an error, maybe we should just never copy the `.git` directory if it exists in a template? Especially since it's not possible to have one when the template is a GitHub repo, which is the primary mechanism for sharing a template.

Reply:
ooh, yeah I like that even better

Reply:
[048ca90](https://github.com/remix-run/remix/pull/7062/commits/048ca90b4e6fadf495421502ecdeb48fb8a5f6bd)